lib/getYtMetadata.ts

```diff
@@ -0,0 +1,8 @@
+'use server';
+export default async function getYtVideoData(videoId: string) {
```

Comment:
can you rename to `youtube`
- shortened name is harder to understand

```suggestion
export default async function getYouTubeVideoData(videoId: string) {
```